transport/TransportImpl.cpp

```diff
@@ -1961,27 +1961,15 @@ void TransportImpl::onIceStateChanged(ice::IceSession* session, const ice::IceSe
 
                 _transportType.store(utils::Optional<ice::TransportType>(endpoint->getTransportType()));
 
-                logger::info("candidate selected %s %s, %s",
+                logger::info("candidate selected %s %s, %s-%s",
                     _loggableId.c_str(),
                     _peerRtpPort.getFamilyString().c_str(),
                     ice::toString(endpoint->getTransportType()).c_str(),
+                    ice::toString(candidatePair.first.type).c_str(),
                     ice::toString(candidatePair.second.type).c_str());
             }
         }
 
```

Comment:
This is the bug fix. This leads to deletion of TcpEndpoint and IceSession still has reference by raw pointer.
We could argue that it is better to remove it from IceSession and then delete it, but strictly we should keep the TCP connectio in case webRTC decides to send over it. One could also argue that is unlikely and wastes resources in SMB in terms of handles, memory and extra load on epoll.